domains.txt

```diff
@@ -362,15 +403,19 @@ jitr.co.uk
 jiyuconsulting.co.uk
 jktech.com
 jmdigital.com
+job-matchers.co.uk
+jobnjob.co.uk
 jobsinscience.com
 jobstheword.co.uk
 jobstheword.com
 jp-engineering.co.uk
 jp-jobs.com
+jpeople.co.uk
 jpng.co.uk
 jps-selection.com
 juicegroup.com
 jumar-recruitment.com
+jumpitsolutions.co.uk
```

Comment:
Roger has been involved in the IT recruitment industry for over 19 years; the last 8 years running Jump IT Recruitment Solutions Ltd. Roger has successfully recruited IT, web and technology staff throughout the UK and also overseas on a permanent, contract and fixed term basis. - See more at: http://www.jumpitsolutions.co.uk/about-us/#sthash.PPbbQYFz.dpuf